app/Http/Controllers/CompetitionsController.php

```diff
@@ -37,7 +47,10 @@ public function create()
      */
     public function store(Request $request)
     {
-        //
+        $this->validate($request, $this->validation_rules);
+        $input = $request->all();
```

Comment:
Since you only use it once, there's no need for this variable – just pass the `$request->all()` directly to `Competition::create`!